installer/test/cloudflowinstance_openshift.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cloudflow
+---
+apiVersion: cloudflow-installer.lightbend.com/v1alpha1
+kind: Cloudflow
+metadata:
+  generation: 1
+  labels:
+    app.kubernetes.io/managed-by: cloudflow-installer
+  name: default
+  namespace: cloudflow
+spec:
+  flink_operator:
+    version: 0.8.2
+    service_account: "some-account"
+  spark_operator:
+    version: 0.6.7
+    image: 1.3.1-OpenJDK-2.4.5-1.1.0-cloudflow-2.12
```

Comment:
Here too I assume

Reply:
These are for testing, so not sure, what to do with these.

Reply:
(not really sure what to do with the test folder, it might be out of date, etc, but we can keep for now)